spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jackson/JacksonProperties.java

```diff
@@ -95,6 +95,11 @@
 	 */
 	private TimeZone timeZone = null;
 
+	/**
+	 * Locale used for formatting.
+	 */
+	private String locale;
```

Comment:
Should be a `Locale` object so that the IDE gets a chance to be smart about the value (no need to act on it, we'll take care of that).